buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_26_0.groovy

```diff
@@ -26,57 +26,14 @@ class GrpcVendoring_1_26_0 {
   static def guava_version = "26.0-jre"
   static def protobuf_version = "3.11.0"
   static def grpc_version = "1.26.0"
-  static def gson_version = "2.8.6"
```

Comment:
I don't understand these deletions

Reply:
analyzeDependencies using wfhartford plugin suggests these dependencies are not required by any module of the Project. So must be removed.